content/20.results.md

```diff
@@ -186,3 +186,28 @@ Estimated composition of name origin prediction over the years of
   (B) For each region, the mean predicted probability of US-affiliated corresponding authors is shown as teal LOESS curve, and the mean probability and 95% confidence interval of the US-affiliated ISCB honoree predictions are shown as dark circles and vertical lines.
 
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/f43f7c40371343a4d8b91438b05c021fdb88af32/figs/us_name_origin.png){#fig:us_name_origin}
+
+### Reception & Feedback
+
+We released [version 1.0](https://github.com/greenelab/iscb-diversity-manuscript/releases/tag/v1.0) of our manuscript on 2020-01-30.
+We submitted this version to the 28^th^ annual ISMB conference planned for July 2020 in Montreal.
+The study received public feedback almost immediately.
+Specifically, a community member [pointed out](https://github.com/greenelab/iscb-diversity-manuscript/issues/27) problems with the country-group names we imported from NamePrism.
+
+On 2020-02-20, ISCB released a statement on equity and inclusion that alluded to our study, but did not directly name or link to it.
+The statment contained the following paragraph:
+
+> We continually revisit our procedures for progress towards equity and inclusion and assess their efficacy.
+> We do not think that automated procedures of assessment are appropriate for this purpose.
+> Algorithms performing such categorization suffer from inherent limitations that distort the notion of balance we are striving for.
+> Specifically, categorization of disjoint regional, ethnic or gender groups of scientists, while easing classification, also functions divisively.
+
+Our ISMB submission was reviewed by 5 peer reviewers and [rejected](https://github.com/greenelab/iscb-diversity-manuscript/issues/37) on 2020-03-13.
+We thank the reviewers for their input, and have made our [responses public](https://github.com/greenelab/iscb-diversity-manuscript/blob/master/content/response-to-reviewers.md)).
+
+Much of the feedback centered around the inherent imperfections of automated detection of race or nationality.
+Therefore, we expanded our assessment to include country of affiliation, which can be detected with high fidelity from PubMed metadata.
+
+We appreciate the feedback we have received.
+Early indications suggest ISCB may be heeding the findings of our analysis and increasing the diversity of honorees.
+TODO: recent ISCB honorees & actions.
```

Comment:
Do we want to run predictions for the 2020 ISMB keynotes & fellows to date and make a small figure for this? PSB happened before our paper came out, but the ISMB stuff should be subsequent to our paper. We could discuss this as early-stage suggestive evidence that an intervention consisting of a public collection and dissemination of such analyses may improve diversity.

Reply:
Yeah that would be cool, either text or figure. Are there any other honorees like 2020 fellows that you think we may have influenced.

Reply:
Perhaps we can merge and address this TODO in a later PR

Reply:
I agree 👍